Forge/src/main/java/me/ferdz/placeableitems/block/PlaceableItemsBlock.java

```diff
@@ -68,6 +69,22 @@ public PlaceableItemsBlock register(String name, Item item, IForgeRegistry<Block
         return this.register(name, registry);
     }
 
+    /**
+     * @see #register(String, IForgeRegistry)
+     */
+    @Deprecated
+    public PlaceableItemsBlock register(String name) {
+        return register(name, GameRegistry.findRegistry(Block.class));
+    }
+
+    /**
+     * @see #register(String, Item, IForgeRegistry)
+     */
+    @Deprecated
+    public PlaceableItemsBlock register(String name, Item item) {
+        return register(name, item, GameRegistry.findRegistry(Block.class));
+    }
+
```

Comment:
If they're no longer used, you don't need to deprecate them. Just remove them

Reply:
Alright. I was thinking more for the sake of compatibility with mods that may add a few blocks, but I'll remove these.